Makefile

```diff
@@ -61,19 +61,19 @@ $(OUT)/trivialDAO_storage.tz : admin_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE
 $(OUT)/trivialDAO_storage.tz : guardian_address = KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh
 $(OUT)/trivialDAO_storage.tz : governance_token_address = KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5
 $(OUT)/trivialDAO_storage.tz : governance_token_id = 0n
-$(OUT)/trivialDAO_storage.tz : now_val = `date +"%s"`
+$(OUT)/trivialDAO_storage.tz : now_level = 100n
```

Comment:
maybe `current_level` would be a better name?

Also, remember to update the `building` doc accordingly as well as the spec.

Reply:
Have changed. Updated `building` doc as well.

Reply:
Oh. It was missing more updates. Have included it now.